allow psutil < 6

jfcg opened the pull request:
fixes autogluon build on MacOS 12.6 and Arm64

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of Apache-2.0 license.

Innixma commented on timeseries/setup.py:
```diff
@@ -27,7 +27,7 @@
     "scipy",
     "pandas",
-    "psutil>=5.7.3,<5.9",
+    "psutil>=5.7.3,<6",
```
`psutil>=5.7.3,<5.10` will be safer. We generally avoid capping at major versions as it is less stable than minor version capping.

shchur replied:
Actually, it seems that we don't use `psutil` anywhere in `timeseries`, so it's fine to remove this dependency completely from `timeseries/setup.py`.

Innixma reviewed (commented):
Thanks for the contribution! Once my comment is addressed, happy to merge. Please also rebase the PR as we fixed a bug in the CI.

Innixma commented:
I updated the PR to reflect my comment, will see if CI passes. I also fixed version range for networkx in timeseries which also had a similar problem.

yinweisu commented:
/platform_tests

jfcg commented:
> I updated the PR to reflect my comment, will see if CI passes. I also fixed version range for networkx in timeseries which also had a similar problem.

Thank you.

shchur reviewed (approved):
LGTM!

jfcg commented:
Hi @tonyhoo, this PR seems to have made it to `v0.6.2` but not mentioned in the release notes, fyi.

Thanks..
